make val_sep useful if label_below == true

TaxiService opened the pull request:
i should've come up with this earlier... 

basically this makes the `val_sep` attribute able to affect the option value's position even if `label_below` is set to true.
meaning that if we set `val_sep` to `-3`, we can do this:

![image](https://user-images.githubusercontent.com/5853667/235343145-17af2b96-a69f-41b6-bf76-adbeeb19a7d9.png)

setting it to `-4` crops the first letter of the option value, but we don't care.

TaxiService commented:
additionally... someday we might want to be able to do this as well
![image](https://user-images.githubusercontent.com/5853667/235343413-60179781-6053-4a62-8efa-e0da7777d8e7.png)

like, have this `val_sep` thing be definable individually for each option. 🤔 idk if we can do this already with some of that _self._ magic or what

myk002 reviewed (commented):
also needs a changelog entry (in the Lua section, I think)

TaxiService commented:
wait- i added wip because i meant to include the attribute name change to "option_gap" in this pr. 🤔 
should i make another one instead? This change is already so small...

...also when you said "needs docs", you mean the changelog?
i didn't include a changelog entry because, like... isn't it weird to say:
"we exposed a new parameter!" and then immediately after
"we changed how the new parameter we just introduced works, slightly!"

...like, why not just merging both changelog entries into the first? 🤔 or is it just too late for that?

silverwing235 commented:
@TaxiService; its because the WIP check is broken somehow, in such a way that it does _not_ like WIPs in titles ATM, and will refuse to complete properly for that reason, as far as some of us can ascertain.

TaxiService commented:
@silverwing235 oh!!! i see. 😳 well, it has been removed now, but still-- consider this kind of a WIP.
Idk if committing the name change or just the changelog... i'll probably wait to hear from myk again before touching anything

myk002 commented:
I think just a small note in the changelog saying that the option name has changed is fine. We don't need to worry about backwards compatibility for something that has existed for such a short time (and really nobody is likely to have used yet)

I think the problem with the WIP in the title is that GitHub itself decided that never-ending actions needed notification emails. The behavior of how the WIP action hasn't changed -- it always just never ended if WIP was in the title -- it's just that GitHub now thinks (as of a few months ago) that that's an error.

The reason that the WIP action doesn't end is because it was coded to loop and detect the eventual absence of "WIP" and then turn green. Nobody is wrong here, it's just a mismatch of expectations.
